pyEpiabm/pyEpiabm/core/_compartment_counter.py

```diff
@@ -85,12 +85,17 @@ def _increment_compartment(self, n_persons: int,
             Status of people being added
         age_group : Age group index
             Person's associated age group
+        add_or_remove : bool
+            True means adding, False will subtract
```

Comment:
I don't find this very intuitive - given the name of the function I think it would be better to support negative integers for n_persons, corresponding to removing people

Reply:
Done